content/events/2022-01-27-tech-industry-night.md

```diff
@@ -0,0 +1,25 @@
+---
+# The title of the event
+title: "Tech Industry Night"
+# Publishing date when the event appears, not the date of the event.
+date: 2022-01-18
+# Tags that apply to the event
+tags: ["career","event","club-activities"]
+# Name of the author (you)
+author: Emilie Ma 
+# Images associated to this event. Used for banner.
+images:
+  - /files/tech-industry-night-2022.jpg
+# Start date and time. Used for calendar page.
+start_date: 2022-01-27 17:00:00
+# End date and time (defaults to one hour after start). Used for calendar page.
+end_date: 2022-01-27 19:00:00
+---
+
+![Tech Industry Night 2022 Banner](/files/tech-industry-night-2022.jpg)
+
+The CSSS will host the first 2022 Tech Industry Night during the inaugural CS week which runs from January 24th-31st.
+
+Students will have the opportunity to interact with industry professionals and recruiters from tech companies and learn more about the job market, interview strategies, resume tips, and more!
+
+Sign up by scanning the QR code on the poster!
```

Comment:
Can you also add this signup survey link?
https://ubc.ca1.qualtrics.com/jfe/form/SV_3a8WDxJVILuRiku